components/x-topic-search/__tests__/x-topic-search.test.jsx

```diff
@@ -43,6 +44,18 @@ describe('x-topic-search', () => {
 		});
 	});
 
+	describe('changed from on blur to on focus', () => {
+		it('should call select on event.target', () => {
```

Comment:
Maybe the following are better descriptions for `describe` and `it` respectively:

* "input receives focus when it already contains a search term"
* "selects the text in the input"

This describes what should be achieved, rather than the technical steps involved.